Makefile

```diff
@@ -1,9 +1,9 @@
 CFLAGS = -Wall -Werror -fpic -std=gnu99
+CC = gcc
```

Comment:
I dont wanna set CC for unix based systems, its not needed, it was only needed for Windows.

Please revert that change

Reply:
Ok, this changes has been reverted :)